storage/src/vespa/storage/distributor/pending_bucket_space_db_transition.cpp

```diff
@@ -11,20 +12,42 @@ LOG_SETUP(".pendingbucketspacedbtransition");
 
 namespace storage::distributor {
 
+using lib::Node;
+using lib::NodeType;
+using lib::NodeState;
+
 PendingBucketSpaceDbTransition::PendingBucketSpaceDbTransition(const PendingClusterState &pendingClusterState,
+                                                               DistributorBucketSpace &distributorBucketSpace,
+                                                               int distributorIndex,
+                                                               bool distributionChanged,
+                                                               const OutdatedNodes &outdatedNodes,
                                                                std::shared_ptr<const ClusterInformation> clusterInfo,
+                                                               const lib::ClusterState &prevClusterState,
```

Comment:
This constructor argument list size is growing outside my comfort zone 😨  We should consider if we can encapsulate this differently during subsequent refactoring.